scripts/ci/datenlord-monitor-test.sh

```diff
@@ -13,8 +11,9 @@ ELASTIICSEARCH_VERSION=${ELASTIICSEARCH_VERSION:-"7.4.2"}
 KIBANA_VERSION=${KIBANA_VERSION:-"7.2.0"}
 
 # Datenlord Monitoring test
-sudo chmod +x scripts/datenlord-monitor-deploy.sh
-cat scripts/alertmanager_test_alert.yaml >> scripts/alertmanager_alerts.yaml
+if ! grep -q "alertmanager_demo" scripts/setup/alertmanager-alerts.yaml; then
+  cat scripts/ci/alertmanager-test-alert.yaml >> scripts/setup/alertmanager-alerts.yaml
+fi
```

Comment:
Shall we merge these two files into a new one? If so, the users don't have to reset the file after the script.